ddl/src/main/resources/templates/kinetica/table/create.sqlt

```diff
@@ -1 +1,6 @@
-CREATE TABLE "[(${schemaName})]"."[(${tableName})]"([(${tableCode})]);
\ No newline at end of file
+[# th:if="${indices} == null or ${indices} == ''"]
```

Comment:
If the `createParams.put("indices", String.join("\n", indicesForTable));` always has a value (indicesForTable is initialised), empty list or something else, we might not need the if/else conditions in this `.sqlt`.